src/KeyVault/Microsoft.Azure.KeyVault/Microsoft.Azure.KeyVault.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Azure.KeyVault</id>
+    <title>Microsoft Azure Key Vault Library</title>
+    <releaseNotes>Initial release</releaseNotes>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/Azure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Provides key and secret operation capabilities for Azure Key Vault.</summary>
+    <description>Provides key and secret capabilities for Azure Key Vault.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft Azure key vault "key vault" REST HTTP client azureofficial windowsazureofficial</tags>
+    <references>
+      <group targetFramework="net45">
+        <reference file="Microsoft.Azure.KeyVault.dll" />
+      </group>
+      <group targetFramework="net40">
+        <reference file="Microsoft.Azure.KeyVault.dll" />
+      </group>
+    </references>
+    <dependencies>
+      <dependency id="Microsoft.Azure.Common" version="[2.0.4,3.0)" />
+      <dependency id="Microsoft.Net.Http" version="2.2.22" />
```

Comment:
My code directly depends on these packages, so it seems like the right thing to do is specify the dependencies here. I understand that since Azure.Common has these dependencies right now, the SDK will work even if I don't specify them in my nuspec, but if Azure.Common dependencies change in some way, my code would break.